090-support-dns-config.md

```diff
@@ -0,0 +1,41 @@
+<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
+
```

Comment:
I guess you can remove this.

Reply:
Addressed in [10ba3f8](https://github.com/strimzi/proposals/pull/142/commits/10ba3f8f70da291a4836d9a7acf0da80c5b30d51)